utils/fo-installdeps

```diff
@@ -172,23 +176,26 @@ if [[ $RUNTIME ]]; then
          case "$CODENAME" in
             xenial)
                apt-get $YesOpt install postgresql-9.5 php7.0 php7.0-pgsql libapache2-mod-php7.0 php7.0-pgsql \
-                 php7.0-cli php7.0-curl php7.0-mbstring php7.0-zip s-nail libboost-program-options1.58.0 \
+                 php7.0-cli php7.0-curl php7.0-mbstring php7.0-zip php-gettext s-nail libboost-program-options1.58.0 \
                  libboost-regex1.58.0 libicu55;;
             stretch)
                apt-get $YesOpt install postgresql-9.6 php7.0 php7.0-pgsql libapache2-mod-php7.0 php7.0-pgsql \
-                 php7.0-cli php7.0-curl php7.0-xml php7.0-zip php7.0-mbstring s-nail libboost-program-options1.62.0 \
+                 php7.0-cli php7.0-curl php7.0-xml php7.0-zip php7.0-mbstring php-gettext s-nail libboost-program-options1.62.0 \
                  libboost-regex1.62.0 libicu57;;
             buster)
                apt-get $YesOpt install postgresql-11 php7.3 php7.3-pgsql libapache2-mod-php7.3 php7.3-pgsql \
-                 php7.3-cli php7.3-curl php7.3-xml php7.3-zip php7.3-mbstring s-nail libboost-program-options1.67.0 \
+                 php7.3-cli php7.3-curl php7.3-xml php7.3-zip php7.3-mbstring php-gettext s-nail libboost-program-options1.67.0 \
                  libboost-regex1.67.0 libicu63;;
             bionic)
                apt-get $YesOpt install postgresql-10 php7.2 php7.2-pgsql libapache2-mod-php7.2 php7.2-pgsql \
-                 php7.2-cli php7.2-curl php7.2-xml php7.2-zip php7.2-mbstring s-nail libboost-program-options1.65.1 \
+                 php7.2-cli php7.2-curl php7.2-xml php7.2-zip php7.2-mbstring php-gettext s-nail libboost-program-options1.65.1 \
                  libboost-regex1.65.1 libicu60;;
             sid)
-               apt-get $YesOpt install postgresql php php-pgsql php-cli php-curl php-xml php-zip php-mbstring s-nail \
+               apt-get $YesOpt install postgresql php php-pgsql php-cli php-curl php-xml php-zip php-mbstring php-gettext s-nail \
                  libboost-program-options1.67.0 libboost-regex1.67.0 libicu63;;
+            focal)
+               apt-get $YesOpt install postgresql php php-pgsql php-cli php-curl php-xml php-zip php-mbstring gettext s-nail \
```

Comment:
The `gettext` package is no longer required here. Just tested on Ubuntu Focal and everything works without it.

Reply:
Okay, I'll remove it